src/constants.js

```diff
@@ -1,75 +1,65 @@
 import * as styledSystem from 'styled-system'
+import { compose } from 'styled-system'
+import systemPropTypes from '@styled-system/prop-types'
+import themeGet from '@styled-system/theme-get'
 import theme from './theme'
 
-const {compose, get: getKey, themeGet} = styledSystem
+const { get: getKey } = styledSystem
 
 export const get = key => themeGet(key, getKey(theme, key))
 
-export const COMMON_LIST = ['color', 'space']
```

Comment:
We used these lists to document system props here: https://github.com/primer/components/blob/master/pages/components/docs/system-props.md

How would you recommend generating these lists for documentation? With the addition of style categories, maybe our system prop groups are simple enough that we can keep them up-to-date manually? @emplums We could potentially link directly to styled-system's reference table https://styled-system.com/table

Reply:
Currently undocumented but each “category” from Styled System has a `.propNames` array, which hopefully you could leverage for this

Reply:
Oh nice. Will that work with `compose` too?

Reply:
Yup. If you're interested in the implementation, the relevant parts are here: https://github.com/styled-system/styled-system/blob/master/packages/core/src/index.js#L68

Each call to `system` uses the `createParser` to create a "parser" function which includes the arguments as a `.config` property – and also adds a `.propNames` property. The `compose` utility uses each function's `.config` property to create a new parser function based on that config. This is intended to be a mechanism not only for internal use, but also as a better way to generate docs for components built with Styled System, and I'd like to add some more docs and utilities for this stuff in the future